modules/ROOT/pages/js-movie-app.adoc

```diff
@@ -122,7 +122,8 @@ Now you’re looking for a demo app or template to start putting those skills in
 
 If you haven't already, clone the https://github.com/neo4j-examples/neo4j-movies-template[Neo4j Movie Template] onto your machine.
 This tutorial post will walk you through rating a movie on a sample movie rating application, from the initial setup to viewing the list of movies you’ve rated.
-The following video will walk briefly walk you through the various features of the movie app. 
+The following video will walk briefly walk you through the various features of the movie app.
+Although the title of the video is "Python App Walkthrough" the functionality of the JavaScript version should be identical. 
```

Comment:
The title of the below video is "Movie App Walkthrough" for me. Maybe remove the warning here?

Reply:
waiting on movies to be updated